### ISSUE TITLE
Use multiple mask per image

### ISSUE THREAD

**Issue by lchauvin** [DAA233/generative-inpainting-pytorch#72]
Hello,

I see in the original paper, they sometimes use multiple mask (or patch) in a single image.

However, I feel like the code is designed for 1 mask / image.

The function random_bbox only return 1 bbox, and there is no dimension to multiple bbox / image.

Did I miss something ?

Thanks

**Comment by daa233** [comment]
You are right! It has been discussed in #31

**Comment by lchauvin** [comment]
In the original paper, however, they show images with multiple regions in the mask (multiple bboxes). Is it done using the same code ?

Thank you.

**Comment by daa233** [comment]
@lchauvin Sorry, this repo is not the official code and I am not the paper's author. The official code is implemented in TensorFlow.

**Comment by lchauvin** [comment]
Indeed, the v1 implementation uses a patch-based computation for the l1 loss, but it seems gone in the v2 for some reason.
